miso-web/src/main/webapp/scripts/list_storage_location.js

```diff
@@ -81,6 +81,9 @@ ListTarget.storage_location = {
         include: true,
         mRender: function (data, type, full) {
           if (type === "display") {
+            if (full.outOfService) {
```

Comment:
This is never set anywhere. Should be added to `StorageLocation` to determine whether out of service (similar to `InstrumentImpl`, but can ignore positions), then also added to `StorageLocationDto` and set in the relevant `Dtos` method so that it gets to the front end